vagrant-spk

```diff
@@ -191,6 +191,72 @@ def call_vagrant_command(sandstorm_dir, *command_args):
     print("Calling {} in {}".format(" ".join(["'{}'".format(arg) for arg in command]), sandstorm_dir))
     return subprocess.check_call(command, cwd=sandstorm_dir)
 
+def ensure_working_vboxsf_in_base_box(sandstorm_dir):
+    # If the .sandstorm/Vagrantfile refers to debian/jessie64, and does not pin it to a particular
+    # version, and the user has version 8.2.2 available on their system, then we abort the
+    # "vagrant-spk up" process because it will not work.
+    def test_vagrantfile_refers_to_jessie64():
+        with open(os.path.join(sandstorm_dir, 'Vagrantfile'), 'r') as fd:
+            for line in fd:
+                if line.strip().split() == 'config.vm.box = "debian/jessie64"'.split():
```

Comment:
The `split()` is supposed to handle some variation, but I can live with people hand-modifying this line losing out, fwiw.

```
>>> 'var = value'.split()
['var', '=', 'value']
>>> 'var    = value'.split()
['var', '=', 'value']
>>> 'var= value'.split()
['var=', 'value']
```

So I guess it's true that I don't catch all variations.

I'll live with this for now. Thank you for noticing it, though; I hadn't thought clearly enough about the variations.